frontend/src/metabase/common/components/QueryColumnPicker/QueryColumnPicker.tsx

```diff
@@ -5,7 +5,8 @@ import { getColumnIcon } from "metabase/common/utils/columns";
 import { Icon, IconName } from "metabase/core/components/Icon";
 import { singularize } from "metabase/lib/formatting";
 
-import * as Lib from "metabase-lib";
+import * as Types from "metabase-lib";
```

Comment:
minor point, but I like this usage. nice to keep types and functional code semantically separate.